### PR TITLE
Add fhir template generation

### PR DESCRIPTION
**Pull request by Thiyan**
## Purpose

> $subject

### CODE REVIEW

File: en/docs/fhir/guides/fhir-resource-api-template.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            For instance:
+
+            * [FHIR base resources](https://central.ballerina.io/ballerinax/health.fhir.r4.international401/latest)
+            * [US Core](https://central.ballerina.io/ballerinax/health.fhir.r4.uscore501/latest)
+            * [AU Base](https://central.ballerina.io/ballerinax/health.fhir.r4.aubase421/latest)
+
+            If you want to generate packages for any custom profiles you can use [Ballerina health package generation tool](https://ballerina.io/learn/health-tool/#package-generation). 
+
+
+3. After you've generated the USCore templates, to run the service as a standalone server, execute the below command.
```
Comment (sameeragunarathne) [inline en/docs/fhir/guides/fhir-resource-api-template.md]: Can you mention that these tempaltes doesn't have the business logic implemented, and it should be implemented by the developer

File: en/docs/fhir/guides/fhir-resource-api-template.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    ```
+    bal health fhir -m template -o genereated_outputs --org-name healthcare_samples --dependent-package ballerinax/health.fhir.r4.uscore501 ig-uscore/definitions
+    ```
+
+    !!! explanaton 
+
+            * -m [mode] - For this instance mode should be ***template***
+            * -o [output directory] - Location of the generated Ballerina artifacts. If this path is not specified, the output will be written to the same directory from which the command is run.
+            * --org-name - [Organization](https://ballerina.io/learn/package-references/#the-org-field) name of the Ballerina package/template to be generated.
+            * --dependent-package - Fully qualified name of the published Ballerina package containing IG resources (eg: <org\>/<package\>). This option can be used to generate templates specifically for the resources in the given IG. The package name part of this value will be added as a prefix to the template name.
```
Comment (sameeragunarathne) [inline en/docs/fhir/guides/fhir-resource-api-template.md]: Let's mention about the dependent package should be prior generated and pushed to  the ballerina central

File: en/docs/fhir/guides/fhir-resource-api-template.md (hunk 1 above)
Comment (Thiyan) [reply en/docs/fhir/guides/fhir-resource-api-template.md]: Done

File: en/docs/fhir/guides/fhir-resource-api-template.md (hunk 2 above)
Comment (Thiyan) [reply en/docs/fhir/guides/fhir-resource-api-template.md]: Done

**Review by sameeragunarathne — APPROVED** [review]

**Pull request merged**